data/data/openstack/bootstrap/variables.tf

```diff
@@ -23,7 +23,36 @@ variable "flavor_name" {
   description = "The Nova flavor for the bootstrap node."
 }
 
-variable "bootstrap_port_id" {
-  type        = string
-  description = "The subnet ID for the bootstrap node."
+variable "enable_bootstrap_floating_ip" {
```

Comment:
Does anything set this variable?  If not, can we drop it?

Reply:
yep, makes sense

Reply:
it was added after the feature freeze, so we couldn't add it to the install-config. now it looks unrelated